### ISSUE TITLE
Shouldn't Week-to-Day conversion be multiplied by 7?

### ISSUE THREAD

**Issue by mykeels** [blake-mealey/ConversionsServer#6]
Here, it's multiplied by 365, which I think is a mistake

https://github.com/blake-mealey/ConversionsServer/blob/c9bfdaf99fecc70adca194b6fc03f2eff5a35628/ConversionsDb/SeedData/Conversions/ConversionData.cs#L44

**Comment by blake-mealey** [comment]
Yes! Feel free to submit a pull request for any data issues you find.
